modules/ROOT/pages/websocket/websocket-connector-reconnection.adoc

```diff
@@ -0,0 +1,32 @@
+= WebSocket Reconnection
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket, reconnection
+
+
+WebSockets are based on persistent and stateful connections. Therefore, it is possible for a connection to be abnormally closed in the lapse between it was established and it's being used in any operation due to factors like idle timeouts, network conditions, etc. 
+
+Inbound sockets cannot be automatically reconnected because for those connections to be reestablished because it is up to the remote system to resend the connection request.
```

Comment:
```suggestion
Inbound sockets cannot be automatically reconnected because the remote system must resend the connection request for the connections to be reestablished.